container/examples/compound-container/index.html

```diff
@@ -0,0 +1,57 @@
+<!-- .js => webcomponent -->
+<!-- .html => iFrame -->
+
+<!-- npm install --save-prod @luigi-project/container --prefix . -->
+
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <title>Luigi Compound Container</title>
+  </head>
+
+  <body>
+    <luigi-compound-container
+      viewURL="./myCompoundWebComponent1.js"
+      id = "compound1"
+      webcomponent="true"
+      context='{"label":"some extra content 1"}'
+    ></luigi-compound-container>
+    
+    <luigi-compound-container
+      viewURL="./myCompoundWebComponent2.js"
+      id = "compound2"
+      webcomponent="true"
+      context='{"label":"some extra content 2"}'
+    ></luigi-compound-container>
```

Comment:
Remove this one. 

Idea is to have one `<luigi-compound-container/>` here which, using the compound config hosts multiple microfrontends inside in a grid format.